MainModule/Server/Core/Functions.lua

```diff
@@ -705,7 +705,7 @@ return function(Vargs, GetEnv)
 			return (gsub(gsub(data, '.', function(x)
 				local r, b = "", byte(x)
 				for i = 8, 1, -1 do
-					r = r..(b % 2 ^ i - b % 2 ^ (i - 1) > 0 and '1' or '0')
+					r ..= (b % 2 ^ i - b % 2 ^ (i - 1) > 0 and '1' or '0')
```

Comment:
This is a thing? Good to know. Guess you can do this crap with any operator then.